test/vmtests/vmtests/test_no_change.py

```diff
@@ -26,3 +37,25 @@ def test_no_change(
         "qcow2",
         output_image_path,
     )
+
+    # Create a differencing disk for the VM.
+    # This will make it easier to manually debug what is in the image itself and what was set during first boot.
+    local_client.run(
+        ["qemu-img", "create", "-F", "qcow2", "-f", "qcow2", "-b", str(output_image_path), str(diff_image_path)],
+    ).check_exit_code()
+
+    # Ensure VM can write to the disk file.
+    os.chmod(diff_image_path, 0o666)
+
+    # Create VM.
+    vm_name = test_instance_name
+    domain_xml = create_libvirt_domain_xml(VmSpec(vm_name, 4096, 4, diff_image_path))
+
+    vm = LibvirtVm(vm_name, domain_xml, libvirt_conn)
+    close_list.append(vm)
+
+    # Start VM.
+    vm.start()
+
+    # Wait for VM to boot by waiting for it to request an IP address from the DHCP server.
+    vm.get_vm_ip_address(timeout=15)
```

Comment:
15 seconds might be too short? (my local tests usually take longer)

Reply:
but it's for iso, so, there are a few more things that take place...

Reply:
In my local testing, I see 6-11 seconds. So, I'll bump this up to 30 seconds.

Reply:
Fixed.